server/app/views/admin/programs/ProgramBaseView.java

```diff
@@ -30,7 +31,13 @@ protected final DivTag renderProgramInfo(ProgramDefinition programDefinition) {
             .withId("program-title")
             .withClasses("text-3xl", "pb-3");
     DivTag description =
-        div(programDefinition.localizedDescription().getDefault()).withClasses("text-sm");
+        div()
+            .with(
+                TextFormatter.formatText(
```

Comment:
This is a new implementation of the formatter to properly format the program description when an admin is editing the screens in a program. ![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/bfe8b4f1-d20b-47cf-ad3a-c68c7354f5e1/9db89ad9-cc7e-4d63-a3ef-485433ae7cc7/Untitled.png)